http4s/client/src/main/scala/endpoints/http4s/client/ChunkedEntities.scala

```diff
@@ -0,0 +1,31 @@
+package endpoints.http4s.client
+
+import org.http4s.{Request => Http4sRequest, Response => Http4sResponse}
+import fs2.Chunk
+
+trait ChunkedEntities
```

Comment:
Awesome! :heart_eyes: 
(we need to add it on the server side then!)

Reply:
I'll probably make a follow up for this soon :)

Reply:
#510